charts/camunda-platform/templates/connectors/_helpers.tpl

```diff
@@ -16,6 +16,14 @@ If release name contains chart name it will be used as a full name.
   {{- include "zeebe.names.gateway" . | replace "\"" "" -}}:{{- index .Values "zeebe-gateway" "service" "gatewayPort" -}}
 {{- end -}}
 
+{{ define "connectors.operateEndpoint" }}
+  {{- print "http://" -}}{{- include "operate.fullname" . | replace "\"" "" -}}:{{- .Values.operate.service.port -}}
+{{- end -}}
+
+{{ define "connectors.keycloakEndpoint" }}
+  {{- print "http://" -}}{{- include "keycloak.fullname" . | replace "\"" "" -}}:{{- .Values.identity.keycloak.service.ports.http -}}
+{{- end -}}
```

Comment:
Hi @aabouzaid,

Need your help here. This one is similar to bellow with `connectors.fullname` but I haven't figured out how to fix this one properly.

Currently it generates `"http://release-name-camunda:80"`.

Do you have any suggestions?

Reply:
@igpetrov it needs the sub-chart object not the main object.
so it should `include "operate.fullname" .Subcharts.operate`.

also, for the template `connectors.keycloakEndpoint`, if you need Keycloak URL, then please use `camundaPlatform.keycloakURL` (it works with default or customized Keycloak depending on the user values file).